pandas/core/generic.py

```diff
@@ -8508,6 +8508,58 @@ def align(
         -------
         (left, right) : ({klass}, type of other)
             Aligned objects.
+
+        Examples
+        --------
+        >>> left = pd.DataFrame([[1,2,3,4], [6,7,8,9]],
+        ...                     columns=['D', 'B', 'E', 'A'],
+        ...                     index=[1,2])
+
+        >>> right = pd.DataFrame([[10,20,30,40], [60,70,80,90], [600,700,800,900]],
+        ...                      columns=['A', 'B', 'C', 'D'],
+        ...                      index=[2,3,4])
+
+        >>> left
+           D  B  E  A
+        1  1  2  3  4
+        2  6  7  8  9
+
+        >>> right
+            A    B    C    D
+        2   10   20   30   40
+        3   60   70   80   90
+        4  600  700  800  900
+
+        >>> a1, a2 = left.align(right,
+        ...                     join='outer',
+        ...                     axis=1)
+
+        >>> a1
+           A  B   C  D  E
+        1  4  2 NaN  1  3
+        2  9  7 NaN  6  8
+
+        >>> a2
+            A    B    C    D   E
+        2   10   20   30   40 NaN
+        3   60   70   80   90 NaN
+        4  600  700  800  900 NaN
+
+        >>> a1, a2 = df1.align(df2,
+        ...                    join='right',
+        ...                    axis=0)
+
+        >>> a1
+            D    B    E    A
+        2  6.0  7.0  8.0  9.0
+        3  NaN  NaN  NaN  NaN
+        4  NaN  NaN  NaN  NaN
+
+        >>> a2
+            A    B    C    D
+        2   10   20   30   40
+        3   60   70   80   90
+        4  600  700  800  900
```

Comment:
these results don't look correct

also, the formatting's off so it won't pass the checks - one way to fix this would be to paste your code in another file, run `black` on it, and then paste it back here. Or paste the entire docstring in another file and run `blackdoc` on it

Reply:
When I run `python scripts/validate_docstrings.py pandas.DataFrame.align` I get these errors wrt example being not correct.
<img width="1377" alt="Screenshot 2021-08-28 at 5 53 45 PM" src="https://user-images.githubusercontent.com/58251592/131217826-8d700f7e-1a9a-4e5d-8779-69bfc39d9311.png">

But when I checked the example in dev version the output differs.
<img width="1377" alt="Screenshot 2021-08-28 at 5 52 59 PM" src="https://user-images.githubusercontent.com/58251592/131217872-f39acb58-1bc4-4b59-b194-580270efb456.png">

PTAL!

Reply:
you've got `join='right'` in your docstring

Reply:
ohh, sry my bad😅